plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -153,7 +152,7 @@ static void installAppEngineRuntime(IFacetedProject project, IProgressMonitor mo
       RuntimeWorkingCopy mutator = (RuntimeWorkingCopy) appEngineRuntimeWorkingCopy;
       Map<String, String> map = mutator.getAttribute(
           "generic_server_instance_properties", new HashMap<>());
-      map.put("cloudSdkDirectory", findCloudSdk());
+      map.put("cloudSdkDirectory", config.getCloudSdkLocation().toString());
```

Comment:
You need to merge with master. I think this part of the code has been removed (and if it hasn't it should be).